stdlib/public/core/FloatingPointTypes.swift.gyb

```diff
@@ -625,7 +625,20 @@ extension ${Self}: BinaryFloatingPoint {
   /// almost never is.
   @_inlineable // FIXME(sil-serialize-all)
   public var ulp: ${Self} {
-    if !isFinite { return ${Self}.nan }
+%if bits == 32 or bits == 64:
```

Comment:
Let's use `bits != 80` for all of these instead of `== 32 or ==64` so that we get the fast implementation for hypothetical future `Float16` / `Float128` automatically.